### PR TITLE
add release notes for v0.19.1

### PR DESCRIPTION
**Pull request by grlee77**
## Description

This is mostly fixes to wheels and some edge cases encountered in the tests on conda-forge, but we did resolve a couple unintentional backwards incompatibile changes.

## Checklist

- [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- Gallery example in `./doc/examples` (new features only)
- Benchmark in `./benchmarks`, if your changes aren't covered by an
  existing benchmark
- Unit tests
- Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)
- Descriptive commit messages (see below)

## For reviewers

- Check that the PR title is short, concise, and will make sense 1 year
  later.
- Check that new functions are imported in corresponding `__init__.py`.
- Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.

### CODE REVIEW

**Comment by psobolewskiPhD** [comment]
Line 4: `We're happy to announce the release of scikit-image v0.19.01`
I think you mean `v0.19.1`?

Also, on macOS 12, M1 python 3.9 and 3.10 wheels from:
https://github.com/grlee77/scikit-image/actions/runs/1557165043
seem fine:
python 3.9.7:
=========== 7152 passed, 41 skipped, 31 warnings in 74.89s (0:01:14) ===========
https://gist.github.com/psobolewskiPhD/25907a88515315e0c5998da8f860fcee

python 3.10.0:
=========== 7152 passed, 41 skipped, 31 warnings in 79.53s (0:01:19) ===========
https://gist.github.com/psobolewskiPhD/26c65900d6062e73b249ff3f1d3a8c13

**Comment by grlee77** [comment]
Thanks @psobolewskiPhD. Most of those are expected warnings, but there are a couple that should be easy to resolve and should have already been raising an error on CI! (we have one test matrix entry that is supposed to raise an error on any unexpected warnings, but clearly that did not happen)

File: doc/release/release_0.19.rst [hunk 1]
```diff
@@ -1,3 +1,37 @@
...
+-------------------
+- Backport PR #6089 on branch v0.19.x (Skip tests requiring fetched data) (gh-6115)
+- Backport PR #6097 on branch v0.19.x (restore non-underscore functions in skimage.data) (gh-6099)
+- Backport PR #6095 on branch v0.19.x (Preserve backwards compatibility for `channel_axis` parameter in transform functions) (gh-6100)
+- Backport PR #6103 on branch v0.19.x (make rank filter test comparisons robust across architectures) (gh-6106)
+- Backport PR #6105 on branch v0.19.x (pass a specific random_state into ransac in test_ransac_geometric) (gh-6107)
+- Fix two equality comparison bugs in the wheel build script (gh-6098)
+- Backport of gh-6109 (Add linker flags to strip debug symbols during wheel building) (gh-6110)
+- Pin setuptools maximum in v0.19.x to avoid breaking on planned distutils API changes (gh-6112)
+- avoid potential circular import of rgb2gray (gh-6113)
```
Comment (mkcor) [inline doc/release/release_0.19.rst:19]:
```suggestion
- Avoid potential circular import of rgb2gray (gh-6113)
```

File: doc/release/release_0.19.rst [hunk 2]
```diff
@@ -1,3 +1,37 @@
+scikit-image 0.19.1
+===================
+
+We're happy to announce the release of scikit-image v0.19.1!
+
+This is a small bug fix release that resolves a couple of backwards compatibility issues and a couple of issues with the wheels on PyPI. Specifically, Apple M1 wheels on PyPI were broken in 0.19.0, but should now be repaired. Wheel sizes are also greatly reduced relative to 0.19.0 by stripping debug symbols from the binaries and making sure that Cython-generated source files are not bundled in the wheels.
+
+
+Other Pull Requests
```
Comment (mkcor) [inline doc/release/release_0.19.rst:9]: Why 'Other?' Isn't this the full list of PRs making up v0.19.1 vs v0.19.0?

Comment (jni) [reply doc/release/release_0.19.rst:9]: It's just the standard heading for things that are uncategorised. Maybe we can rename it to "Uncategorized Pull Requests"? :shrug: I don't think it's that important...

Comment (grlee77) [reply doc/release/release_0.19.rst:9]: That header was autogenerated by the `tools/generate_release_notes.py` script. It does seem odd to list it as "other" though. It is the full list.

Comment (mkcor) [reply doc/release/release_0.19.rst:9]: Sure, no big deal, I was just curious. I think the header could write "Pull Requests included" or simply "Pull Requests..."

File: doc/release/release_0.19.rst [hunk 3]
```diff
@@ -1,3 +1,37 @@
+scikit-image 0.19.1
+===================
+
+We're happy to announce the release of scikit-image v0.19.1!
+
+This is a small bug fix release that resolves a couple of backwards compatibility issues and a couple of issues with the wheels on PyPI. Specifically, Apple M1 wheels on PyPI were broken in 0.19.0, but should now be repaired. Wheel sizes are also greatly reduced relative to 0.19.0 by stripping debug symbols from the binaries and making sure that Cython-generated source files are not bundled in the wheels.
```
Comment (mkcor) [inline doc/release/release_0.19.rst:6]:
```suggestion
This is a small bug fix release that resolves a couple of backwards compatibility issues and a couple of issues with the wheels on PyPI. Specifically, OSX M1 wheels on PyPI were broken in 0.19.0, but should now be repaired. Wheel sizes are also greatly reduced relative to 0.19.0 by stripping debug symbols from the binaries and making sure that Cython-generated source files are not bundled in the wheels.
```
(more standard)

**Review by mkcor — COMMENTED** [review]
Do we want to wait for #6118 to be merged and/or #6093 to be decided upon to release v0.19.1? If not, I'm happy to approve.

File: doc/release/release_0.19.rst (hunk 3 above)
Comment (psobolewskiPhD) [reply doc/release/release_0.19.rst:6]: Good catch, maybe make it consistent with the wheel arch too so: Apple M1 (arm64)

Comment (psobolewskiPhD) [reply doc/release/release_0.19.rst:6]: Actually, I think it's important to mention the M1 wheel is for macOS 12 (Monterey):
`export MACOSX_DEPLOYMENT_TARGET=12.0`
It's been out for nearly 2 mo, but probably not everyone runs it.

Comment (grlee77) [reply doc/release/release_0.19.rst]: Thanks, the language has been updated to mention this

**Comment by grlee77** [comment]
should be ready now. everything else tagged for 0.19.1 has been merged so once this is in I can kick off the release

**Review by hmaarrfk — APPROVED** [review]

File: doc/release/release_0.19.rst [hunk 4]
```diff
@@ -1,3 +1,42 @@
...
+- Backport PR #6089 on branch v0.19.x (Skip tests requiring fetched data) (gh-6115)
+- Backport PR #6097 on branch v0.19.x (restore non-underscore functions in skimage.data) (gh-6099)
+- Backport PR #6095 on branch v0.19.x (Preserve backwards compatibility for `channel_axis` parameter in transform functions) (gh-6100)
+- Backport PR #6103 on branch v0.19.x (make rank filter test comparisons robust across architectures) (gh-6106)
+- Backport PR #6105 on branch v0.19.x (pass a specific random_state into ransac in test_ransac_geometric) (gh-6107)
+- Fix two equality comparison bugs in the wheel build script (gh-6098)
+- Backport of gh-6109 (Add linker flags to strip debug symbols during wheel building) (gh-6110)
+- Pin setuptools maximum in v0.19.x to avoid breaking on planned distutils API changes (gh-6112)
+- Avoid potential circular import of rgb2gray (gh-6113)
+- Backport PR #6089 on branch v0.19.x (Skip tests requiring fetched data) gh-6115
```
Comment (mkcor) [inline doc/release/release_0.19.rst:20]:
```suggestion
- Backport PR #6089 on branch v0.19.x (Skip tests requiring fetched data) (gh-6115)
```
to be consistent with all other lines

**Review by mkcor — APPROVED** [review]
